app/impl/delete_route.go

```diff
@@ -19,7 +19,7 @@ func (s *DeleteRouteService) New() RequestImpl {
 
 // DeleteRouteRequest is delete route request
 type DeleteRouteRequest struct {
-	RouteID *int `json:"route_id,omitempty"`
+	SeqNo int `json:"seq_no"`
```

Comment:
```suggestion
	SeqNo *int `json:"seq_no, omitempty"`
```

@oshikun 
`omitempty`は空の場合、もしくは変数がセットされていない場合にNullをいれるためのプロパティです。

Nullを許容しているのはGormを使用している都合上、nilで入ってきた場合に条件として認識されないためユーザーデータを全て削除してしまうため、Nilの場合を防ぐためにNilの可能性を考慮しています。